scss/_variables.scss

```diff
@@ -499,6 +499,8 @@ $input-transition:                      border-color .15s ease-in-out, box-shado
 
 $form-text-margin-top:                  .25rem !default;
 
+$form-grid-gutter-width:                10px !default;
```

Comment:
I would move this line to where `$form-group-margin-bottom` is set, those variables are a bit related.